frontend/src/components/Main/OptionsForm/OptionsFormList.js

```diff
@@ -16,6 +16,11 @@ class OptionsFormList extends Component {
             optionsList = toJS(optionsStore.optionsList);
         return (
             <div>
+                <div className="text-danger">
+                    {optionsStore.hasLogNormalSelected
```

Comment:
I think we should keep this text all the time, instead of only when selected. Could you use the `HelpTextPopup` component in the table header?